XenAdmin/Wizards/NewSRWizard_Pages/Frontends/CIFS_ISO.cs

```diff
@@ -165,9 +166,22 @@ public string SrDescription
 
         private void textBoxCifsSharename_TextChanged(object sender, EventArgs e)
         {
+            passwordFailure1.PerformCheck(IsSRNameUnique);
+
             OnPageUpdated();
         }
 
+        private bool IsSRNameUnique(out string error)

```

Comment:
Could we please change this method name to a more relevant name? Because this method actually checks if the iso is already attached , not if the name is unique.